tests/Makefile

```diff
@@ -1,132 +1,239 @@
--include .local.make
-DOCKER_IMAGE_NAME := docker.io/zenko/zenko-e2e
-DOCKER_IMAGE_TAG := latest
-
-DOCKER := docker
-TOX := tox
+# Some bin aliases
+DOCKER := @docker
+TOX := @tox
+HELM := @helm
 KUBECTL := kubectl
-HELM := helm
-
-DOCKER_IMAGE := $(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)
 
 DOCKER_ARGS :=
 PYTEST_ARGS :=
 DOTENV := $(PWD)/.env
-HELM_TEST_PODS := {range .items[?(@.metadata.annotations.helm\.sh\/hook)]}{@.metadata.name}{"\n"}{end}
-E2E_POD := zenko-e2e
+
+# These are for dumping logs, but they're really long so store them as variables
+E2E_PODS := {range .items[?(@.metadata.annotations.helm\.sh\/hook)]}{@.metadata.name}{"\n"}{end}
```

Comment:
Could this var be confused with `E2E_POD` below as they don't seem to serve the same purpose?